join.py

```diff
@@ -0,0 +1,17 @@
+import sys
+from csv_joiner import Joiner
+
+
+def main():
+    try:
+        args = ['D:/projekty python/virtus-lab-task/files/user_device.csv', 'D:/projekty python/virtus-lab-task/files/user_usage.csv']
```

Comment:
Bonus question, if it was a sensitive information committed here, how would you remove it from a) git b) github?
```suggestion
```